Automattic/newspack-newsletters
# feat(ads): make auto ad insertion optional

### All Submissions:






### Changes proposed in this Pull Request:

1205078412187623-as-1205078412187697

Implements the `disable_auto_ads` newsletter meta and deprecates `diable_ads`.

### How to test the changes in this Pull Request:

1. While on the master branch, draft a newsletter and toggle `Disable ads for this newsletter` on
2. Check out this branch and edit the drafted newsletter
3. Confirm the new "Ads Settings" panel with the `Disable automatic insertion of ads` toggled on
4. The panel should also render information about existing ads:
<img width="288" alt="image" src="https://github.com/Automattic/newspack-newsletters/assets/820752/86f71850-5667-43a2-9b8d-77deb3489fe4">

6. Make sure you have Newsletter Ads
7. Send the newsletter and confirm no ads are rendered
8. Draft a new newsletter with the option toggled off and send
9. Confirm ads render

### Other information:
